<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidgetpp">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageproduit">
       <widget class="QStackedWidget" name="stackedwidgetprincipale">
        <property name="geometry">
         <rect>
          <x>199</x>
          <y>10</y>
          <width>871</width>
          <height>651</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="pageajoutproduit">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <widget class="QPushButton" name="bouttonajouterproduit">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>360</y>
               <width>181</width>
               <height>29</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
             </property>
             <property name="text">
              <string>ajouter</string>
             </property>
            </widget>
            <widget class="QFrame" name="frame_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>60</y>
               <width>181</width>
               <height>281</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Type</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Nom</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Prix</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Quantité</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QFrame" name="frame_4">
             <property name="geometry">
              <rect>
               <x>210</x>
               <y>60</y>
               <width>221</width>
               <height>281</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QComboBox" name="boxtype">
                <item>
                 <property name="text">
                  <string>naturel</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>chimique</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputnomp">
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
    border: 1px solid #64B5F6; /* Bleu clair */
    border-radius: 3px;
    padding: 5px;
    background-color: white;
}
QLineEdit:focus {
    border: 2px solid #1E88E5; /* Bleu vif */
}</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputprixp">
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
    border: 1px solid #64B5F6; /* Bleu clair */
    border-radius: 3px;
    padding: 5px;
    background-color: white;
}
QLineEdit:focus {
    border: 2px solid #1E88E5; /* Bleu vif */
}</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputquantite">
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
    border: 1px solid #64B5F6; /* Bleu clair */
    border-radius: 3px;
    padding: 5px;
    background-color: white;
}
QLineEdit:focus {
    border: 2px solid #1E88E5; /* Bleu vif */
}</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pagemodifierproduit">
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>60</y>
            <width>121</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>ID Produit</string>
          </property>
         </widget>
         <widget class="QFrame" name="frame_5">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>120</y>
            <width>161</width>
            <height>281</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Nom</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Prix</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Quantité</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="frame_6">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>120</y>
            <width>191</width>
            <height>281</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLineEdit" name="inputtypemodi">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    border: 1px solid #64B5F6; /* Bleu clair */
    border-radius: 3px;
    padding: 5px;
    background-color: white;
}
QLineEdit:focus {
    border: 2px solid #1E88E5; /* Bleu vif */
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="inputnommodif">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    border: 1px solid #64B5F6; /* Bleu clair */
    border-radius: 3px;
    padding: 5px;
    background-color: white;
}
QLineEdit:focus {
    border: 2px solid #1E88E5; /* Bleu vif */
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="inputprixmodif">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    border: 1px solid #64B5F6; /* Bleu clair */
    border-radius: 3px;
    padding: 5px;
    background-color: white;
}
QLineEdit:focus {
    border: 2px solid #1E88E5; /* Bleu vif */
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="inputquantitemodif">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    border: 1px solid #64B5F6; /* Bleu clair */
    border-radius: 3px;
    padding: 5px;
    background-color: white;
}
QLineEdit:focus {
    border: 2px solid #1E88E5; /* Bleu vif */
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLineEdit" name="idproduitmodifie">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>70</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 1px solid #64B5F6; /* Bleu clair */
    border-radius: 3px;
    padding: 5px;
    background-color: white;
}
QLineEdit:focus {
    border: 2px solid #1E88E5; /* Bleu vif */
}</string>
          </property>
         </widget>
         <widget class="QPushButton" name="bouttonidproduitmodif">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>70</y>
            <width>83</width>
            <height>29</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
          </property>
          <property name="text">
           <string>envoyer</string>
          </property>
         </widget>
         <widget class="QPushButton" name="bouttonconfirmermodification">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>420</y>
            <width>83</width>
            <height>29</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
          </property>
          <property name="text">
           <string>Confirmer</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="pageafficherproduits">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTableView" name="tableauproduits">
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_pdf">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
            </property>
            <property name="text">
             <string>PDF</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_7">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="inputrecherchetableau">
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border: 1px solid #64B5F6; /* Bleu clair */
    border-radius: 3px;
    padding: 5px;
    background-color: white;
}
QLineEdit:focus {
    border: 2px solid #1E88E5; /* Bleu vif */
}</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bouttonrecherchetableau">
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
               </property>
               <property name="text">
                <string>Rechercher</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="triparnom">
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
               </property>
               <property name="text">
                <string>tri par prix</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="validationsms">
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
               </property>
               <property name="text">
                <string>Validation SMS</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pagestatistique">
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>80</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>statistique</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="pagearduino">
         <widget class="QLineEdit" name="lineEditEmployeeID">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>100</y>
            <width>171</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonverifer">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>100</y>
            <width>83</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>verifer</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelResult">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>320</y>
            <width>321</width>
            <height>91</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="pagesupprimerproduit">
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>80</y>
            <width>151</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>ID Produit</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="idproduitsupprime">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>110</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 1px solid #64B5F6; /* Bleu clair */
    border-radius: 3px;
    padding: 5px;
    background-color: white;
}
QLineEdit:focus {
    border: 2px solid #1E88E5; /* Bleu vif */
}</string>
          </property>
         </widget>
         <widget class="QPushButton" name="bouttonsupprimerproduit">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>110</y>
            <width>83</width>
            <height>29</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="pagehistorique">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTextEdit" name="textHistorique"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>192</width>
          <height>641</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget {
    background-color: #E3F2FD; /* Bleu clair */
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="bouttonpageafficherproduit">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
           </property>
           <property name="text">
            <string>Afficher Produits</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bouttonpagestatistique">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
           </property>
           <property name="text">
            <string>Statistique des produits</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bouttonpagemodifierproduit">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
           </property>
           <property name="text">
            <string>Modifier Produit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bouttonpagesupprimer">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
           </property>
           <property name="text">
            <string>Supprimer Produit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bouttonpageajouter">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
           </property>
           <property name="text">
            <string>Ajouter Produit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bouttonpagehistorique">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #2196F3; /* Bleu primaire */
    color: white;
    border: 1px solid #0D47A1; /* Bleu foncé */
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #1976D2; /* Bleu intermédiaire */
}
QPushButton:pressed {
    background-color: #1565C0; /* Bleu plus foncé */
}</string>
           </property>
           <property name="text">
            <string>Historique</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bouttonpagearduino">
           <property name="text">
            <string>Arduino</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonexit">
           <property name="text">
            <string>exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="menup">
       <widget class="QPushButton" name="pushButtonkhedmakol">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>301</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>produit</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>301</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>client</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>230</y>
          <width>301</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>machine</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
